src/snapshot_middleware/mod.rs

```diff
@@ -388,3 +392,58 @@ pub fn default_sync_rules() -> &'static [SyncRule] {
         ]
     })
 }
+
+fn filter_default_property(
+    snapshot: &SyncbackSnapshot,
+    new_inst: &Instance,
+    name: &str,
+    value: &Variant,
+    attributes: &mut BTreeMap<String, UnresolvedValue>,
+    properties: &mut BTreeMap<String, UnresolvedValue>,
```

Comment:
Many parameters :(

Is there a way this could be simplified?

Reply:
6 parameters is... a lot, but I can't think of a way to reduce it without complicating logic somewhere else. I think it's probably fine as-is though, since the call sites look okay.